address_book/internal/config/config.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	"github.com/caarlos0/env/v6"
```

Comment:
I think, it's better to use the flags instead of the environment variables: you may use different types (not only the strings), it's possible to display the help message with supported flags, it's easier to manipulate with params.

Reply:
ok, i will use flags for the following tasks, but to my mind, this is a controversial issue: "Which is better: env vars or flags ?"

Reply:
I agree with you, this question may cause a huge holywar, and you can leave the existing implementation as is. But in our projects, we usually deal with flags, not with environment variables